docs/src/modules/components/overview/charts/mainDemo/PieChartDemo.tsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable material-ui/no-hardcoded-labels */
+import * as React from 'react';
+import { PieChart } from '@mui/x-charts/PieChart';
+import Typography from '@mui/material/Typography';
+
+const data = [
+  { label: 'Group A', value: 400 },
+  { label: 'Group B', value: 300 },
+  { label: 'Group C', value: 300 },
```

Comment:
Should we use more meaningful data? E.g., v5, v6, v7?

Reply:
I've changed it to @mui/material version split on 8th Jun-25. It's a bit random but still not so random. Open to suggestions.